src/ts/background/services/keyring-vault.ts

```diff
@@ -6,15 +6,19 @@ import { mnemonicGenerate, cryptoWaitReady, mnemonicValidate } from '@polkadot/u
 import t from '../../services/i18n'
 import { SimpleAccounts, MessageExtrinsicSign } from '../types'
 import { createType } from '@polkadot/types'
+import { VALIDITY_INTERVAL } from '../../constants/config'
 
 const VAULT_KEY: string = 'speckle-vault'
 
 class KeyringVault {
 
   private _keyring?: KeyringInstance
   private _password?: string
+  private _tempPassword?: string
   private _mnemonic?: string
   private simpleAccounts?: SimpleAccounts
+  private _expiryTimeLeft = 0
+  private _expiryTimeLeftId = 0
```

Comment:
_expiryTimeLeftId -> _expiryTimeLeftTimer?

Reply:
I think `_expiryTimeLeftTimerId` makes more sense since it's the `timerId` that is being returned by the `setInterval` function. Let me know what you think. Thanks!

Reply:
yup.